patches/minecraft/net/minecraft/entity/EntityLivingBase.java.patch

```diff
@@ -39,7 +39,27 @@
      }
  
      public EntityLivingBase func_110144_aD()
-@@ -812,6 +814,8 @@
+@@ -668,8 +670,17 @@
+         else
+         {
+             Collection<PotionEffect> collection = this.field_70713_bf.values();
+-            this.field_70180_af.func_187227_b(field_184634_g, Boolean.valueOf(func_184593_a(collection)));
+-            this.field_70180_af.func_187227_b(field_184633_f, Integer.valueOf(PotionUtils.func_185181_a(collection)));
++            boolean hideParticles = Boolean.valueOf(func_184593_a(collection));
++            int potionColor = Integer.valueOf(PotionUtils.func_185181_a(collection));
```

Comment:
It was boxed in the decompiled code, they just copied it. Not a big problem imo.